### PR TITLE
update aicoeci base image

### PR DESCRIPTION
**Pull request by goern**
- added some test cases
- update AICoE-CI image version
- update pre-commit plugins
- update pyproject

### CODE REVIEW

File: features/data/project/provenance_flask_error/Pipfile [hunk 1]
```diff
@@ -7,4 +7,4 @@ name = "pypi"
 flask = "*"
 
 [requires]
-python_version = "3.8"
+python_version = "3.9"
```
Comment (fridex) [inline features/data/project/provenance_flask_error/Pipfile:10]:
```suggestion
python_version = "3.8"
```

File: .pre-commit-config.yaml [hunk 2]
```diff
@@ -1,12 +1,12 @@
 ---
 repos:
   - repo: git://github.com/Lucas-C/pre-commit-hooks
-    rev: v1.1.9
+    rev: v1.1.13
```
Comment (codificat) [inline .pre-commit-config.yaml:4]: Please take the opportunity to switch from `git://` to `https://`  (ref https://github.com/thoth-station/thoth-application/issues/2111)

File: .pre-commit-config.yaml [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
     hooks:
       - id: remove-tabs
 
   - repo: git://github.com/pre-commit/pre-commit-hooks
-    rev: v3.4.0
+    rev: v4.1.0
```
Comment (codificat) [inline .pre-commit-config.yaml:9]: same here (switch from `git://` to `https://`), but also there is another `pre-commit/pre-commit-hooks` section below, so better fold the hooks into that one (and remove duplicates)

**Review by codificat — CHANGES_REQUESTED** [review]
A couple of requests for pre-commit config updates

**Comment by mayaCostantini** [comment]
I think this PR can be closed as changes were already made by https://github.com/thoth-station/integration-tests/pull/304 and https://github.com/thoth-station/integration-tests/pull/307

**Pull request closed without merging**